src/ansys/fluent/core/fluent_connection.py

```diff
@@ -55,6 +57,21 @@ def run(self) -> None:
             cb()
 
 
+def get_container_ids() -> List[str]:
+    try:
+        logger.debug("Attempting to get docker container IDs...")
+        proc = subprocess.Popen(
```

Comment:
@raph-luc 
Some time subsequent to this PR, would you be interested in switching the code to use the Python docker package?